compiler-k2/src/main/kotlin/com/squareup/anvil/compiler/k2/util/AnvilNames.kt

```diff
@@ -0,0 +1,10 @@
+package com.squareup.anvil.compiler.k2.util
+
+import com.squareup.anvil.annotations.ContributesBinding
+import com.squareup.anvil.annotations.ContributesTo
+import com.squareup.anvil.compiler.internal.fqName
+
+internal object AnvilNames {
+  val contributesBinding = ContributesBinding::class.fqName
+  val contributesTo = ContributesTo::class.fqName
```

Comment:
The entire `:compiler-utils` dependency is being added for the sake of this `fqName` extension and the `joinSimpleNames` extension.  Whatever API we wind up using for defining the `FqName`s, it would be good if we could just define the values using strings.